ocaml/utils/zero_alloc_utils.ml

```diff
@@ -247,3 +247,10 @@ module Assume_info = struct
 
   let is_none t = match t with No_assume -> true | Assume _ -> false
 end
+
+let is_check_enabled ~opt =
```

Comment:
It's not obvious to me why this has been added.  I think its only call site is in `Selectgen` and the function that used to be called there still exists.

Reply:
Good catch, thanks! It's a stray copy of `Builtin_attributes.is_check_enabled` that has been renamed to `is_zero_alloc_check_enabled`. I moved it out of `Lambda` and into `Zero_alloc_utils` before the rebase over the signatures.